### PR TITLE
[BUG] Fix of MicroXSReader precheck mechanics

### PR DESCRIPTION
**Pull request by aalfonsi**
Please add the appropriate tag to the beginning of the title of your PR:

1. `[DOC]` if documentation related
1. `[DEP]` if something is deprecated or a deprecated feature is removed
1. `[ENH]` if a feature or enhancement
1. `[BUG]` if a bug fix
1. `[API]` if a (possibly incompatible) change to the API
1. `[TST]` if related to our test suite
1. `[STY]` if related to code cleanliness and pythonicness

For bug fixes and enhancements, link to the corresponding issue.

Closes https://github.com/CORE-GATECH-GROUP/serpent-tools/issues/464

Make sure you have read over the developer guide to ease
the review process. These include:

- [ ] PR fits the [project scope](http://serpent-tools.readthedocs.io/en/latest/develop/contributing.html#project-scope)
- [ ] Code to be committed is [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and matches the [code style](http://serpent-tools.readthedocs.io/en/latest/develop/codestyle.html#code-style) of the project
- [ ] New and/or changed features are [well documented](http://serpent-tools.readthedocs.io/en/latest/develop/documentation.html#documentation) and have adequate testing
- [ ] For first-time contributors, you have included your attribution in [`docs/contributors.rst`](https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/develop/docs/contributors.rst)

Include a thorough and concise overview of the changes, and why this PR is overall beneficial to the project.

Once the PR has been created and is nearing closure, make sure that serious changes, including deprecations and incompatible changes are documented in [the changelog](https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/develop/docs/changelog.rst)

### CODE REVIEW

File: serpentTools/parsers/microxs.py [hunk 1]
```diff
@@ -182,7 +182,7 @@ def _precheck(self):
                               "Parser: {}\nFile: {}".format(self,
                                                             self.filePath))
             for tline in fid:
-                if 'NFY' in tline:
+                if 'NFY' in tline or 'XS_' in tline:
                     return
             raise SerpentToolsException("Fission yields values were not "
                                         "found in {}".format(self.filePath))
```
Comment (drewejohnson) [inline serpentTools/parsers/microxs.py:188]: Only recommendation would be changing the exception here to reflect missing fission yields or cross sections. But I'll defer to review and potential approval by @DanKotlyar

Comment (DanKotlyar) [reply serpentTools/parsers/microxs.py:188]: I think this change does make sense,

Comment (drewejohnson) [reply serpentTools/parsers/microxs.py:188]: @aalfonsi bump

Comment (aalfonsi) [reply serpentTools/parsers/microxs.py]: addressed comment.

**Review by drewejohnson — APPROVED** [review]
Thanks @aalfonsi 

Pinging @DanKotlyar

**Comment by DanKotlyar** [comment]
Thanks @aalfonsi 
@drewejohnson - should I take any action?

**Comment by drewejohnson** [comment]
@DanKotlyar I'm not sure why py 3.9 isn't going in testing. But once that gets resolved you should be able to merge this in

**Comment by drewejohnson** [comment]
@aalfonsi can you change the source branch from `develop` to `master`? We don't have py3.9 testing on `develop` - 

https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/f61cd104bd997aa80429f1059bbc3669adc18654/.github/workflows/testing.yaml#L20

**Comment by aalfonsi** [comment]
Unfortunately if I do, this PR adds ~90 files with several conflicts:

Screen Shot 2022-10-14 at 12 22 18 PM

**Comment by aalfonsi** [comment]
> Unfortunately if I do, this PR will add ~90 files with several conflicts:
> 
> Screen Shot 2022-10-14 at 12 22 18 PM
> 
> Probably the best solution (if master is the real "default" branch) is for me to apply these changes in a new branch from master. What do you think?

Oh never mind...you said it :) "source branch" not "target branch". No problem...I can do it.

**Comment by aalfonsi** [comment]
[microxs_patch.txt](https://github.com/CORE-GATECH-GROUP/serpent-tools/files/9789497/microxs_patch.txt)

Since the "master" branch is neither a protected branch or a default branch, it does not show up in my fork. The only way to contribute to master for me is to create a patch file (as attached).

**Comment by DanKotlyar** [comment]
@drewejohnson How should I proceed with this?

**Comment by aalfonsi** [comment]
> @drewejohnson How should I proceed with this?

Since the modification is quite small, Can one of you apply the patch in a new branch (based on master) and then we can close this MR ?

**Comment by drewejohnson** [comment]
@aalfonsi sorry for the confusion. I'll take your patch and create a new PR off `master`

@DanKotlyar can you take a look at these comments - https://github.com/CORE-GATECH-GROUP/serpent-tools/issues/466#issuecomment-1279286356 This should help resolve some confusion going forward

**Pull request closed without merging**